eng/targets/Npm.Common.targets

```diff
@@ -18,7 +18,7 @@
 
   <Target Name="Restore">
     <Message Importance="High" Text="Running yarn install on $(MSBuildProjectFullPath)" />
-    <Yarn Command="install $(InstallArgs)" StandardOutputImportance="High" StandardErrorImportance="High" />
+    <Yarn Command="install --mutex network $(InstallArgs)" StandardOutputImportance="High" StandardErrorImportance="High" />
```

Comment:
Is there a reason we're using `network` instead of `file`? It seems like `network` is more likely to get us in to port conflict issues...

Reply:
I tried file first, thinking the same thing, but that failed right away with
```
YARN : error : Unable to update lock within the stale threshold [F:\workspace\_work\1\s\src\SignalR\clients\ts\common\common.npmproj]
```

Google didn't turn up a useful answer for what causes this.

Reply:
Yeah, I was thinking that using the file mutex with an absolute location (like `/tmp/.yarn-mutex`) would be the most reliable.

Reply:
Ok. We can see how `network` goes :).

Reply:
Let's use the scientific method. Hypothesis - file mutex is more reliable.

Data so far:
* file mutex - 13 successful PR checks, 2 failures
* network mutex - 15 successful PR checks, 0 failures

I can trigger more runs before merging if we don't want to experiment with this on the master branch.

Reply:
No need for more tests, I just didn't refresh the page before you said you'd tested it. If you've got data that trumps my hunch :)

Reply:
![SCIENCE!](https://media.giphy.com/media/z6ccg9ZZzWT2E/giphy.gif)